src/components/10-atoms/radio/demo.js

```diff
@@ -81,36 +81,45 @@ storiesOf('Atoms/Radio/Demos', module)
 </svg>`;
 
     const template = html`
-    <axa-fieldset horizontal>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 1"
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 2"
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 3"
-        checked
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 4"
-        disabled
-        .icon="${icon}"
-    ></axa-radio>
-    </axa-fieldset>
-`;
+      <axa-fieldset horizontal>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 1"
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 2"
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 3"
+          checked
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 4"
+          disabled
+          .icon="${icon}"
+        ></axa-radio>
+      </axa-fieldset>
+    `;
 
     render(template, root);
     return root;
-  });
+  })
+  .add(
+    'Radio - no label',
+    () => `
+      <axa-fieldset>
+        <p>This axa-radio's have no labels. Therefore no label will be rendered</p>
+        <axa-radio id="radio1" name="insurance"></axa-radio>
+        <axa-radio id="radio2" name="insurance" label=""></axa-radio>
```

Comment:
Does this need an extra story?

Reply:
I need it to test it...

Reply:
You can test individually with knobs. Please have a look if you can build a knob for that option.

Reply:
The problem is, that I can't pass an empty label with the knobs. This results in the default label. In our case `Ja`.

Reply:
Are you sure that this is not possible? `&knobname=&somethingelse=hi`

Reply:
only works when there is no default value set, but I need a default value :(

Reply:
As discussed with raphi in the daily, it is not possible in this case. Therefore I will close this conversation.